app/js/services/situationService.js

```diff
@@ -187,8 +187,16 @@ angular.module('ddsApp').factory('SituationService', function($http, $sessionSto
 
         flattenPatrimoine: flattenPatrimoine,
 
+        getDemandeur: function(situation) {
+            return _.find(situation.individus, { role: 'demandeur' });
+        },
+
         hasEnfantScolarise: function(situation) {
             return !!_.find(situation.individus, { role: 'enfant', scolarite: 'college' }) || !!_.find(situation.individus, { role: 'enfant', scolarite: 'lycee' });
+        },
+
+        hasEnfant: function(situation) {
+            return !!_.find(situation.individus, { role: 'enfant' });
```

Comment:
So in JS, as long as the code is fine with it, you shouldn't cast ? 
The fact that the function name `hasEnfant` suggests it returns a `boolean` is not considered a problem ?